test/PowerShellEditorServices.Test/Utility/AsyncQueueTests.cs

```diff
@@ -70,6 +70,9 @@ public async Task AsyncQueueSkipsCancelledTasks()
             cancellationSource.Cancel();
             await inputQueue.EnqueueAsync(1);
 
+            // Wait for things to propegate.
+            await Task.Delay(1000);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4993607)